manifests/webclient_root.pp

```diff
@@ -1,26 +1,50 @@
-class i2b2::webclient_root(
+class i2b2::webclient_root inherits i2b2::params
+{
+  $domains =
+  [
+    {
+      domain => 'i2b2demo',
+      name => 'HarvardDemo',
+      urlCellPM => 'http://services.i2b2.org/i2b2/services/PMService/',
+      allowAnalysis => true,
+      debug => false,
+    },
+  ]
 
-) inherits i2b2::params {
-  require i2b2
+  $webclient_dir = "$intermediary_files_dir/i2b2webclient-$version"
+  $webclient_zip = "$webclient_dir.zip"
 
-  $webclient_zip = "$intermediary_files_dir/i2b2webclient-$version.zip"
+  class {'::apache' :
+    default_vhost => false,
+    mpm_module    => 'prefork',
+  }
+  class {'::apache::mod::php' : }
+  apache::vhost { 'localhost':
+    port    => 80,
+    docroot => "$webclient_dir/webclient",
+  }
```

Comment:
Apache configuration should go into a profile, otherwise module re-usability is reduced.